keyboards/orthograph/readme.md

```diff
@@ -0,0 +1,17 @@
+# Orthograph
```

Comment:
This could do with following https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md slightly more closely.